<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>244</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tabla de Alumnos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alumno</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Correo</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UltimoAcceso</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Actualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Importar">
       <property name="text">
        <string>Importar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Eliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_Buscar"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Buscar">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
